src/content/blog/2024_mitiq_impact.md

```diff
@@ -0,0 +1,147 @@
+
+<style>
+  .side-by-side {
+    display: flex;
+    justify-content: center; /* Center align the images */
+    gap: 10px; /* Add spacing between images */
+  }
+  .side-by-side img {
+    max-width: 100%;
+    height: auto;
+    border-radius: 8px; /* Optional rounded corners */
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1); /* Optional shadow */
+  }
+  .side-by-side div {
+    text-align: center; /* Center captions under images */
+    width: 45%; /* Control image width */
+  }
+</style>
+
+<div class="side-by-side">
+  <div>
+    <img src="/images/2024-mitiq/qermit.png" alt="Bar chart of the Qermit downloads per year from 2021 to 2024.">
+    <p>Qermit saw a 100% increase in downloads YoY!</p>
+  </div>
+  <div>
+    <img src="/images/2024-mitiq/pygsti.png" alt="Bar chart of the pyGSTi downloads per year from 2016 to 2024.">
+    <p>pyGSTi amassed over 120k downloads in a calendar year!</p>
+  </div>
+</div>
+
+Two takeaways from these plots are
+
+1. QEM as a whole is growing, not just Mitiq, and
+2. lots of people are running tomography experiments.
+
+These two facts, taken together with the lack of tooling[^1] to take noise characterization data to inform QEM showcase a gap in our ecosystem.
+A gap that we hope to (at least start to) fill in the coming year.
+
+[^1]: IBM has recently released the a new API: [`NoiseLearner`](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.noise_learner.NoiseLearner). This is an excellent example of using noise characterization to inform runtime, and should be generalized to work across the quantum stack (not just Qiskit).
+
+In addition to noise characterization, we've seen two other needs from our community.
+
+1. On the software side, tooling is needed for QEM technique comparison and combination, and
+2. on the research side, the industry needs a better understanding of how QEM will interplay with QEC.
+
+We hope to make progress on both of these in 2025, and we won't be able to without your help!
+We need fresh ideas, people writing code, doing research, giving talks, etc.
+If this sounds like something you're interested in, I encourage you to get involved.
+
+## Join the Mitiq Community
+
+Whether you’re a researcher, educator, or software engineer, or quantum tinkerer, there’s a place for you in the Mitiq community.
+
+- Check out the [`mitiq`](https://github.com/unitaryfund/mitiq) GitHub repository and contribute by opening issues, asking questions, or making a pull request. If you have a QEM technique you'd like to use, let us know! Feature requests are always appreciated, and encouraged.
+- Join our [Discord community](http://discord.unitary.fund) to connect with me and all the other wonderful Mitiq developers and quantum enthusiasts.
+- Sign up to recieve the [Mitiq Newsletter](https://forms.gle/6UcUjSawHyweXhQV7) to stay up to date with the latest happenings.
```

Comment:
I am going to add this call to action at the bottom of the newsletter as well. Someone may forward the newsletter to a friend/colleague and, in turn, they may want to sign up.